components/lib/inputnumber/InputNumber.js

```diff
@@ -40,6 +40,7 @@ export const InputNumber = React.memo(
         const _minusSign = React.useRef(null);
         const _currency = React.useRef(null);
         const _decimal = React.useRef(null);
+        const _decimalSeparator = React.useRef(null);
```

Comment:
i think we should add a comment next to each field about what they hold like...

```js
const _decimal = React.useRef(null); // regular expression /[,]/g, /[.]/g, /[]/g
const _decimalSeparator = React.useRef(null); // ?????

Reply:
Do these need to be commented together?
```js
const _minusSign = React.useRef(null);
const _currency = React.useRef(null);
const _decimal = React.useRef(null);
```

Reply:
That would be great.